package_cliartifacts.sh

```diff
@@ -20,12 +20,14 @@ pkg_tar() {
     x86_64)
 		  dir=linux
     	mkdir "${OUTDIR}/${dir}"
-    	mv kn-linux-amd64 ${OUTDIR}/${dir}/kn
+      mv kn-linux-amd64 ${OUTDIR}/${dir}/kn
+      chmod u+x ${OUTDIR}/${dir}/kn
```

Comment:
Maybe I'd do just `chmod +x`. It feels more common to add exec to all and copy to `/usr/bin` etc as root.

@rhuss wdyt?

Reply:
changed to `chmod +x`